api/swagger.yaml

```diff
@@ -2260,7 +2260,9 @@ definitions:
                     ConfigName is the name of the config that this references, but this is just provided for
                     lookup/display purposes. The config in the reference will be identified by its ID.
                   type: "string"
-
+          Isolation:
```

Comment:
oh boy, you're right; It looked as if it was at the wrong indentation level, but you're right. Sorry, my mistake ha!